packages/react/src/components/Nav/Nav.base.tsx

```diff
@@ -386,6 +386,9 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
   }
 
   private _isGroupExpanded(group: INavLinkGroup): boolean {
```

Comment:
Since this is modified to always return `group.isCollapsed` when defined, should we also explicitly not call `_toggleCollapsed` from within `_onGroupHeaderClicked` when `isCollapsed` is controlled?

Right now calling it means we're checking `_isGroupExpanded` twice in succession and updating the component state when it's guaranteed to be unnecessary.